ios/Sources/TimetableFeature/TimetableDetailScreen.swift

```diff
@@ -0,0 +1,245 @@
+                        placeholderSize: .medium,
+                        width: Const.imageSize,
+                        height: Const.imageSize
+                    )
+                    .clipShape(Circle())
+                    .overlay(
+                        RoundedRectangle(cornerRadius: Const.imageSize / 2)
+                            .stroke(AssetColor.Separate.image.color, lineWidth: 1)
+                    )
+                    Text(speaker.name)
+                        .font(.body)
+                        .foregroundColor(AssetColor.Base.secondary.color)
+
+                    Spacer()
+                }
+            }
+        }
+        .padding(.bottom, 24)
+    }
+
+    func descriptions(timetable: TimetableItem) -> some View {
+        VStack(alignment: .leading, spacing: 6) {
+            ForEach(DesciptionType.allCases, id: \.self) { type in
+                HStack(spacing: 11) {
+                    type.image
+                        .foregroundColor(AssetColor.Base.primary.color)
+                        .frame(width: Const.iconSize, height: Const.iconSize)
+
+                    Text(type.value(timetable: timetable))
+                        .font(.footnote)
+                        .foregroundColor(AssetColor.Base.secondary.color)
+
+                    Spacer()
+                }
+            }
+        }
+        .padding(8)
+        .background(AssetColor.Background.secondary.color)
+        .cornerRadius(8)
+        .padding(.bottom, 32)
+    }
+
+    func body(timetable: TimetableItem) -> some View {
+        VStack(alignment: .leading, spacing: 32) {
+            VStack(alignment: .leading, spacing: 8) {
+                Text(L10n.TimetableScreen.Detail.intendedAudience)
+                    .font(.body)
+                    .bold()
+                    .foregroundColor(AssetColor.Base.primary.color)
+                Text("ViewModel、LiveDataぐらいはわかるが、StateFlow辺りから少し混乱している方") // TODO: FIXME
+                    .font(.body)
+                    .foregroundColor(AssetColor.Base.secondary.color)
+                    .lineLimit(nil)
+            }
+
+            VStack(alignment: .leading, spacing: 8) {
+                Text(L10n.TimetableScreen.Detail.description)
+                    .font(.body)
+                    .bold()
+                    .foregroundColor(AssetColor.Base.primary.color)
+                Text("みなさんは、AndroidのUIの関連の状態管理や非同期処理について、CoroutiensのrepeatOnLifecycleまでそのメリット、デメリットまで含めて追えていますか？以下のQiitaではAndroidのライフサイクルの基礎からKotlin Coroutinesでのsuspend functionによる処理までは紹介しました。https://qiita.com/takahirom/items/3f012d46e15a1666fa33しかし、現在はKotlin Coroutines Flow、StateFlow、今はさらにlaunchWhenStarted、repeatOnLifecycleが登場して、少し混乱しているのではないでしょうか？ここまでの流れをまとめて紹介していきたいです。") // TODO: FIXME
+                    .font(.body)
+                    .foregroundColor(AssetColor.Base.secondary.color)
+                    .lineLimit(nil)
+            }
+        }
+        .padding(.bottom, 84)
```

Comment:
Is this needed?

Reply:
Yes. when scrolling to the bottom, we cannot scroll to the end because we have fixed footer in this view.